packages/elements/src/components/icon/icon.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import { shallow, ShallowWrapper, ReactWrapper, mount } from "enzyme";
+import React from "react";
+import { act } from "react-dom/test-utils";
+import { Icon } from "./icon";
+
+describe("Icon", () => {
+  describe("constructor", () => {
+    let wrapper: ReactWrapper;
+
+    beforeEach(() => {
+      act(() => {
+        wrapper = mount(<Icon name="tick" />);
+      });
+    });
+
+    it("should render an svg element at root", () => {
+      expect(wrapper.getDOMNode().tagName).toBe("svg");
+    });
+    it("should render a use element as first child", () => {
+      expect(wrapper.find("use").length).toEqual(1);
+    });
+
+    afterEach(() => {
```

Comment:
Having `afterEach` before all the `it` blocks will be more readable. Usually, what goes it `afterEach` usually has something to do with what we do in `beforeEach`.

Reply:
Good point, thanks!